cmd/ocsp-updater/main.go

```diff
@@ -354,7 +367,12 @@ func (updater *OCSPUpdater) findRevokedCertificatesToUpdate(batchSize int) ([]co
 	var statuses []core.CertificateStatus
 	_, err := updater.dbMap.Select(
 		&statuses,
-		`SELECT * FROM certificateStatus
+		`SELECT
```

Comment:
@jsha for this PR would you like to see me move the list of fields to select into a constant in `sa/model.go`?